blueoil/cmd/main.py

```diff
@@ -68,10 +68,27 @@ def init(output):
     '--experiment_id',
     help='ID of this training.',
     default=None,
-    required=False,
 )
-def train(config, experiment_id=None):
-    experiment_id, checkpoint_name = run_train(config, experiment_id)
+@click.option(
+    '--recreate',
+    is_flag=True,
+    help='Delete and recreate experiment id dir',
+    default=False,
+)
+@click.option(
+    '-n',
+    '--network',
+    help='Network name which you want to use for this training. override config.DATASET_CLASS',
```

Comment:
I don't know why we need this option.
Specify in config file is not enough?

Reply:
I also don't know why but I think we can remove this.

Reply:
I've removed the options `network` and `dataset`.